_specs_/websocket/schemas/forex/events/aggregates.yml

```
@@ -0,0 +1,21 @@
+type: object
+properties:
+  ev:
+    type: string
+    enum:
+      - CA
+  pair:
+    type: string
+    description: The currency pair.
+  o:
+    $ref: '../../index.yml#/AggsOpen'
+  c:
+    $ref: '../../index.yml#/AggsClose'
+  h:
+    $ref: '../../index.yml#/AggsHigh'
+  l:
+    $ref: '../../index.yml#/AggsLow'
+  v:
+    $ref: '../../index.yml#/AggsVolume'
+  s:
+    $ref: '../../index.yml#/Timestamp'
```

Forex aggs also have an `e` end timestamp
```suggestion
  s:
    $ref: '../../index.yml#/Timestamp'
  e:
    $ref: '../../index.yml#/Timestamp'
```